src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Message.java

```diff
@@ -1,10 +1,11 @@
 package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
 
+import java.util.List;
+import java.util.Optional;
 import lombok.Getter;
 import lombok.Setter;
 
-import java.util.List;
-
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
```

Comment:
Do we need this suppression? Not sure I like it specifically mentioned in code. Unless we can do it somewhere else, I'd say we remove it, and just ignore the highlighting of `Optional` in IntelliJ

Reply:
Generally speaking I'd still prefer us to avoid using `Optional` fields, as per API recommendations and industry best practice. This way we're calling it out as an exception to the rule.

Reply:
It'd be nicer if we could include it in `config/checkstyle/suppressions.xml` (so it's not in our implementation), but if it's a pain then I'm happy with this